web/src/containers/IncentivePayments.js

```diff
@@ -45,32 +43,32 @@ class IncentivePayments extends Component {
                   <th id={thId('null2')} />
                   <Fragment key={year}>
                     {QUARTERS.map(q => (
-                      <th key={q} className="right-align" id={thId(year, q)}>
+                      <th key={q} className="right-align" id={thId(year, q)} scope="col">
```

Comment:
Since we're not using the `headers` attr anymore, can we also get rid of the `id`s on these header cells?